packages/react-widgets/package.json

```diff
@@ -66,7 +66,8 @@
     "webpack-cli": "^4.5.0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.13.9"
+    "@babel/runtime": "^7.13.9",
```

Comment:
As we pretend to deprecate this package, I prefer not to dig into this too much, but I think this dependency shouldn't be here 🤦